Userland/Applications/Calculator/main.cpp

```diff
@@ -70,6 +69,22 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         widget->set_entry(Crypto::BigFraction { Crypto::SignedBigInteger(16180339887), power });
     }));
 
+    auto& round_menu = window->add_menu("&Round");
+    GUI::ActionGroup preview_actions;
+
+    static constexpr Array<StringView, 3> numbers_label { "zero"sv, "two"sv, "four"sv };
```

Comment:
This is a minor, but I think it would feel more natural if these were "0", "2", and "4"? Using the full words for numbers seemed a tad off, IMO.

```c++
    static constexpr auto rounding_modes = Array { 0, 2, 4 };

    for (auto rounding_mode : rounding_modes) {
        auto round_action = GUI::Action::create_checkable(String::formatted("To &{} digits", rounding_mode), [&widget, rounding_mode](auto&) {
            widget->set_rounding_length(rounding_mode);
        });

        preview_actions.add_action(*round_action);
        round_menu.add_action(*round_action);
    }
```